src/constants/tokens.ts

```diff
@@ -0,0 +1,15 @@
+import { arbitrum, base, mainnet, optimism, polygon } from 'viem/chains';
+
+import { CosmosChainId } from './graz';
+
+export const USDC_ADDRESSES = {
+  [mainnet.id]: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
```

Comment:
are these safe to hard code?

Reply:
ya these are pretty much hard coded on circle's website too - https://developers.circle.com/stablecoins/usdc-on-main-networks